packages/twenty-server/src/engine/api/graphql/workspace-schema-builder/graphql-types/input/array-filter.input-type.ts

```diff
@@ -0,0 +1,10 @@
+import { GraphQLInputObjectType, GraphQLList, GraphQLString } from 'graphql';
+
+export const ArrayFilterType = new GraphQLInputObjectType({
```

Comment:
what is this used for?

Reply:
without this filter type the graphql throws an error